packages/livecd/live-boot/package/boot/uefi/loader/entries/cos-x86_64.conf

```diff
@@ -1,4 +1,4 @@
 title cOS Linux Live
 version x86_64
 efi /minimal/x86_64/kernel.xz
-options initrd=/minimal/x86_64/rootfs.xz cdroot rootdelay=7
+options initrd=/minimal/x86_64/rootfs.xz root=live:/dev/cdrom rd.live.dir=/ rd.live.squashimg=rootfs.squashfs
```

Comment:
This `/dev/cdrom` parameter can be avoided if we set a LABEL to the ISO, I think it is the `-volid` parameter of `xorriso`.

Reply:
I've added the support for it over weekend, should be enough to set `label: "foo"` in the `iso/cOS-local.yaml` file

Reply:
Awesome, gonna test it :)